src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,126 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static RegistrationService registrationService;
+    private User user1;
+    private User user2;
+    private User user3;
+
+    @BeforeAll
+    static void beforeAll() {
+        registrationService = new RegistrationServiceImpl();
+    }
+
+    @BeforeEach
+    void setUp() {
+        user1 = new User();
+        user1.setLogin("im_the_first");
+        user1.setPassword("first_pass");
+        user1.setAge(33);
+
+        user2 = new User();
+        user2.setLogin("im_the_second");
+        user2.setPassword("second_pass");
+        user2.setAge(18);
+
+        user3 = new User();
+        user3.setLogin("im_the_third");
+        user3.setPassword("third_pass");
+        user3.setAge(19);
+    }
+
+    @AfterEach
+    void tearDown() {
+        Storage.people.clear();
+    }
+
+    @Test
+    void registration_nullUser_notOk() {
+        user1 = null;

```

Comment:
Please, rewrite your tests according to the rules:
given -> new User() with parameters
when -> register method (in some cases  assertThrows(RegistrationException.class, () -> registrationService.register(user));
then -> assertEquals 

You should initialise test data in each test method